### PR TITLE
Player controller implementation

### PR DESCRIPTION
**Pull request by alex-deaconu**
@ezequielc @settinghead please review. Thanks!

### CODE REVIEW

File: src-js/Viewer/Player/RisePlayerController.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  }
+};
+  
+var _pingPlayer = function () {
+  if (isActive) {
+    var url = baseUrl + cmdHeartbeat + "?callback=?";
+    _callUrlNative(url, "ping");
+    _pingV3Watchdog();
+
+    setTimeout(_pingPlayer, HEARTBEAT_TIME);
```
Comment (settinghead) [inline src-js/Viewer/Player/RisePlayerController.js]: This is related to the comment above. I think having a global `setInterval` would suffice. If for some reason _pingPlayer() needs to be called on an ad-hoc basis (say as soon as the player is set to 'active'), it should just execute the function which also ensures one and only one execution loop is present. So maybe something like this:

```js
var pingLoop;

var _pingPlayer = () => {
  if (isActive) {
     ... 
  }
  if(!pingLoop) {
    pingLoop = setInterval(_pingPlayer, HEARTBEAT_TIME);
  }
};
```

**Comment by alex-deaconu** [comment]
@settinghead good call. Will fix. Thanks!

File: src-js/Viewer/Player/RisePlayerController.js (hunk 1 above)
Comment (alex-deaconu) [inline src-js/Viewer/Player/RisePlayerController.js]: @settinghead updated to use the suggested method. Please see second commit. Thanks!

**Comment by settinghead** [comment]
@alex-deaconu all good now. :shipit:

**Comment by ezequielc** [comment]
@alex-deaconu 👍

**Pull request merged**